examples/demo13.py

```diff
@@ -93,11 +93,11 @@ def main(argv):
     logger.debug('Read in WFIRST imaging filters.')
 
     logger.info('Reading from a parametric COSMOS catalog.')
-    # Read in a galaxy catalog - just a random subsample of 100 galaxies for F814W<23.5 from COSMOS.
+    # Read in a galaxy catalog - just a random subsample of 100 galaxies for F814W<25.2 from COSMOS.
```

Comment:
It essentially boils down to whether the example catalog file containing
100 galaxies that this demo loads was changed when the COSMOSCatalog was
updated. I have changed the comments in few other places too to say that we
get galaxies brighter than 25.2.
I think the maximum magnitude in the example catalog comes out to be under
22.5, so it really doesn't make the difference.

On Wed, May 4, 2016 at 1:21 PM, Rachel Mandelbaum <notifications@github.com>
wrote:

> In examples/demo13.py
> <https://github.com/GalSim-developers/GalSim/pull/744#discussion_r62078875>
> :
>
> > @@ -93,11 +93,11 @@ def main(argv):
> >      logger.debug('Read in WFIRST imaging filters.')
> >
> >      logger.info('Reading from a parametric COSMOS catalog.')
> > -    # Read in a galaxy catalog - just a random subsample of 100 galaxies for F814W<23.5 from COSMOS.
> > +    # Read in a galaxy catalog - just a random subsample of 100 galaxies for F814W<25.2 from COSMOS.
>
> Oh I thought you meant in this demo (that the catalog was being
> initialized using a sample name). I see what you mean now.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/GalSim-developers/GalSim/pull/744/files/356cf2f039aa391c0c6c6a2a1f9093d50ef29507#r62078875>
>



-- 
Regards

Arun Kannawadi Jayaraman
PhD candidate, Dept. of Physics,
Carnegie Mellon University,
Pittsburgh, PA 15213